tslint.json

```diff
@@ -10,6 +10,9 @@
   },
   "defaultSeverity": "warning",
   "rules": {
-    "prettier": true
+    "prettier": true,
+    "interface-over-type-literal": false,
+    "no-console": false,
+    "class-name": false
```

Comment:
Minor: You shouldn't have these as `false` - they're good lint rules, especially the `no-console` one.
`class-name` can be customised to allow `_` prefixes.